R/R/allocator.R

```diff
@@ -133,7 +133,8 @@ robyn_allocator <- function(robyn_object = NULL,
 
   ## set local variables, sort & prompt
   # paid_media_spends <- InputCollect$paid_media_spends
-  paid_media_selected <- InputCollect$paid_media_selected
+  paid_media_selected <- if ("paid_media_selected" %in% names(InputCollect))
+    InputCollect$paid_media_selected else InputCollect$paid_media_spends
```

Comment:
For this change, would you please try simply replacing `InputCollect$paid_media_selected` with `InputCollect$paid_media_vars`. If that works, I'd prefer that. Otherwise looks good.

Reply:
Ok, so for the one-pagers we can use `InputCollect$paid_media_vars` directly, no need to check if it exists because it's legacy. What is new is `paid_media_selected`. Changing to `InputCollect$paid_media_vars` then and getting rid of this check.

Reply:
@gufengzhou checked and working with legacy results as well.